circle.yml

```diff
@@ -35,7 +35,7 @@ commands:
             else
               [[ $OSTYPE = darwin* ]] && os=macos || os=ubuntu
               cd /usr/local
-              curl -L https://github.com/WebAssembly/wabt/releases/download/1.0.20/wabt-1.0.20-$os.tar.gz | sudo tar xz --strip 1
+              curl -L https://github.com/WebAssembly/wabt/releases/download/1.0.20/wabt-1.0.20-$os.tar.gz | (sudo tar xz --strip 1 || tar xz --strip 1)
```

Comment:
This solves the problem of `sudo` not working in Rust docker (user is already root there), probably there are more elegant ways to solve it, but this works.

Reply:
@gumb0 @chfast Since #706 also runs into this problem with `sudo pip3`, how about we just install sudo on the rust tasks and leave these helpers intact?

Reply:
It might be better to migrate to [cimg/rust](https://circleci.com/developer/images/image/cimg/rust) where build are probably not executed as root and there may be more CI related benefits.